src/main/scala/com/microsoft/hyperspace/index/IndexConstants.scala

```diff
@@ -56,6 +56,12 @@ object IndexConstants {
   // See https://github.com/microsoft/hyperspace/issues/185
   val INDEX_RELATION_IDENTIFIER: (String, String) = "indexRelation" -> "true"
 
+  // Tags are used while applying rules.
+  // INDEX_HYBRIDSCAN_REQUIRED_TAG indicates if Hybrid Scan is required for this index or not.
+  // This is set in getCandidateIndexes and utilized in transformPlanToUseIndex.
+  val INDEX_HYBRIDSCAN_REQUIRED_TAG: IndexLogEntryTag[Boolean] =
+  IndexLogEntryTag[Boolean]("hybridScanRequired")
+
```

Comment:
This file is getting out of control with random constants. Can you create `IndexLogEntryTags` and put tag related constants there? I should be able to use as `IndexLogEntryTags.HYBRID_SCAN_REQUIRED`, for example.

Reply:
@imback82 I updated #164 based on this PR. Thanks!

Reply:
@imback82 Could you check the change and merge this change? Thanks!